### PR TITLE
Add 209 handwritten ocr notebook

### PR DESCRIPTION
**Pull request by Debskij**

### CODE REVIEW

**Comment by helena-intel** [comment]
@Debskij the CI fails on Windows on the function that tests for README's. Could you look into that? Maybe the `test_readme()` function needs to be adjusted.

**Comment by ryanloney** [comment]
At first glance it looks good. I shared some feedback with Kuba this morning, it would be nice to make it easier to swap out a custom image. At some point optional use of the webcam (like 400 series notebooks) would be nice as well. 

I tried uploading a row of simplified Chinese symbols and it works with just one duplicated symbol. I asked Wenjing if symbols will ever be duplicated in actual handwritten text (if not, this simplifies the post-processing code)
image

**Comment by ryanloney** [comment]
@Debskij  please show how to use the Japanese text model as well, since this notebook is meant to include both models. If the threshold needs to be adjusted, we should show that in the notebook with the sample image here: https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/handwritten_text_recognition_demo/python/data/handwritten_japanese_test.png

**Comment by adrianboguszewski** [comment]
I'm ok with that, as I already reviewed this before. And of course, it works for me (tested Windows and Ubuntu).

@helena-intel I could only suggest storing small image files in the repository instead of downloading them from the Internet. Those files should have just about a few hundred kB.

**Review by adrianboguszewski — APPROVED** [review]

**Review by ryanloney — APPROVED** [review]

File: notebooks/209-handwritten-ocr/209-handwritten-ocr.ipynb [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1d3b1190",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Select language by using either language='chinese' or language='japanese'\n",
+    "language = 'japanese'\n",
```
Comment (helena-intel) [inline notebooks/209-handwritten-ocr/209-handwritten-ocr.ipynb:110]:
```suggestion
    "language = 'chinese'\n",
```
I suggest Chinese as default language

File: notebooks/209-handwritten-ocr/209-handwritten-ocr.ipynb [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+   },
+   "outputs": [],
+   "source": [
+    "import cv2\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from collections import namedtuple\n",
+    "from itertools import groupby\n",
+    "from openvino.inference_engine import IECore\n",
+    "from os import path"
```
Comment (helena-intel) [inline notebooks/209-handwritten-ocr/209-handwritten-ocr.ipynb:41]:
```suggestion
    "from pathlib import Path"
```
I changed the calls that use this too. I would like to standardize on using pathlib.

**Comment by helena-intel** [comment]
@Debskij there is a conflict with notebooks/README.md. Can you rebase/merge to main and fix that?

**Comment by helena-intel** [comment]
I fixed an issue that I introduced with a PR suggestion (sorry for that) and autoformatted the file (with jupyterlab-code-formatter) which also sorts imports according to PEP8.

**Review by helena-intel — APPROVED** [review]

**Comment by helena-intel** [comment]
There still are two requests for small clarifications/changes but they are not blocking and I will merge the notebook now. Can you make the changes in a new PR @Debskij ? That's not urgent.

**Comment by jnzw** [comment]
FYI, there's an English handwritten OCR model to be released soon to the OMZ:
https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/handwritten-english-recognition-0001

**Pull request merged**